src/node_quic_socket.cc

```diff
@@ -555,7 +555,7 @@ void QuicSocket::SendVersionNegotiation(
   if (req->Send() != 0) delete req;  // TODO(addaleax): Better error handling?
 }
 
-ssize_t QuicSocket::SendRetry(
+bool QuicSocket::SendRetry(
```

Comment:
Is this change being used somewhere? It conflicts with what I’m currently doing :)

Reply:
Like, I can mirror these changes, it’s not an issue, it just would be good to know if there’s more to it than I’m seeing, because as of right now the return value is unused afaict

Reply:
Simplifying and making things consistent. Generally, I'm trying to get rid of as many of the ssize_t uses as possible to simplify error handling.

Reply:
But in general, we may be able to just make this a void return because we're not currently using the return value at all